man/systemd-creds.xml

```diff
@@ -353,6 +353,14 @@
         be possible without losing access to all previously encrypted secrets.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-certificate=</option><arg>PATH</arg></term>
+
+        <listitem><para>Takes a path to a PEM encoded X509 certificate. If specified,
+        <command>systemd-creds</command> will bind the encryption to the certificate's public key. This
+        option cannot be used together with <option>--tpm2-public-key</option>.</para></listitem>
```

Comment:
suffix `--tpm2-public-key` with `=` please

hmm, wouldn't it be nicer to just make the existing `--tpm2-public-key=` key switch to also read a certificate if one is passed? Sounds simpler to me, then having two distinct options.

i.e. try to parse as cert and if it doesn't, parse as public key? there should not be any ambiguity possible